library/src/main/java/com/panoramagl/hotspots/PLHotspot.java

```diff
@@ -247,10 +247,19 @@ public void getRect(PLRect rect) {
 
     @Override
     public void setX(float x) {
+        if(this.getX() != x)
+		{
+			super.setX(x);
+			hasChangedCoordProperty = true;
```

Comment:
The codestyle is ugly. Because the codestyle is checked-in, a simple format will do the job 
![image](https://user-images.githubusercontent.com/3314607/119128173-96256c00-ba35-11eb-8698-ea47fd872b6b.png)

Reply:
Yeah i know, I just modified from GitHub and it formatted it that way.
However I will fix it

Reply:
Did it.